File: src/test/java/com/ject/studytrip/trip/fixture/ConfirmTripReportImageRequestFixture.java
package com.ject.studytrip.trip.fixture;

import com.ject.studytrip.trip.presentation.dto.request.ConfirmTripReportImageRequest;

public class ConfirmTripReportImageRequestFixture {
    private String tmpKey = "tmp/trip-reports/1/test.jpg";

    public ConfirmTripReportImageRequest build() {
        return new ConfirmTripReportImageRequest(tmpKey);
    }

    public ConfirmTripReportImageRequestFixture withTmpKey(String tmpKey) {
        this.tmpKey = tmpKey;
        return this;
    }
}

File: src/test/kotlin/com/ject/studytrip/trip/fixture/ConfirmTripReportImageRequestFixture.kt
package com.ject.studytrip.trip.fixture

import com.ject.studytrip.trip.presentation.dto.request.ConfirmTripReportImageRequest

class ConfirmTripReportImageRequestFixture(
    private val tmpKey: String = "tmp/trip-reports/1/test.jpg",
) {
    fun withTmpKey(tmpKey: String): ConfirmTripReportImageRequestFixture = ConfirmTripReportImageRequestFixture(tmpKey)

    fun build(): ConfirmTripReportImageRequest = ConfirmTripReportImageRequest(tmpKey)
}
